RadixWallet/Clients/OverlayWindowClient/OverlayWindowClient+Interface.swift

```diff
@@ -113,10 +102,34 @@ extension OverlayWindowClient {
 			}
 		}
 
+		public struct Icon: Hashable, Sendable {
+			public enum Kind: Hashable, Sendable {
+				case asset(ImageAsset)
+				case system(String)
+			}
+
+			public let kind: Kind
+			public let foregroundColor: Color
+
+			public init(
+				kind: Kind,
+				foregroundColor: Color = .app.green1
+			) {
+				self.kind = kind
+				self.foregroundColor = foregroundColor
+			}
+		}
+
 		case hud(HUD)
 		case alert(AlertState)
+		case sheet(SheetOverlayCoordinator.Root.State, SheetBehavior)
 		case fullScreen(FullScreenOverlayCoordinator.State)
 	}
+
+	public enum SheetBehavior: Sendable {
```

Comment:
Is this to be used to replace the old sheet content when tapping on the embeded info link? if yes, why isn't the replacement simply done in the InfoLinkSheet view?

Reply:
sure, but in the context of info sheet, can't the replace happen internally? what is the reason for having the replace to happen in Overlay reducer? seems like logic specific to the info sheet

Reply:
> Ah, you mean that we could do the replacing right inside the info sheet without even involving the OverlayReducer at all?

Yup, that would simplify things.

> One reason I made it like this is that I imagine other situations where we might want to replace the existing sheet rather than enqueueing, for example if we move the Completion sheets to the Overlay.

This enqueueing or replacing should probably be a parameter to the scheduling function itself, or higher level item, as it is not specific to the sheet.